roles/sap_swpm/tasks/post_install.yml

```diff
@@ -26,8 +26,8 @@
       - '   SAP SWPM deployment successfully completed '
       - ' '
       - '   SAP Product        -       {{ sap_swpm_product_catalog_id }}  '
-      - '   SID                -       {{ sap_swpm_sid }} '
-      - '   Primary Instance   -       {{ sap_swpm_pas_instance_nr }} '
+      - '   SID                -       {{ sap_swpm_sid | d("") }} '
+      - '   Primary Instance   -       {{ sap_swpm_pas_instance_nr | d("") }} '
```

Comment:
```suggestion
      - '   Primary Instance   -       {{ sap_swpm_pas_instance_nr | d('') }} '
```

Reply:
@Wabri As discussed above, using nested quotes of the same time will not work. So we'll leave the double quotes here.

Reply:
Yes, I mark this as resolve